examples/jsm/nodes/core/NodeBuilder.js

```diff
@@ -192,6 +210,12 @@ class NodeBuilder {
 
 	}
 
+	getMIPLevelAlgorithmNode( textureNode, levelNode ) {
```

Comment:
Why we need this as a method of `NodeBuilder`?

Reply:
`SamplerLevel` can be controlled in [ 0, 1 ] values, for this worked in PBR is necessary an algorithm, I'll remove it from here and move to `EnvironmentNode` as `Context` and use just a linear approach for non-pbr until I finish this PR.

https://github.com/mrdoob/three.js/pull/25041/files#diff-b3dec78c4d4656e9b96d66830127da312a15734c78494a3717e5c3a7d26a5320